docs/usage/configuration-options.md

```diff
@@ -4312,6 +4312,15 @@ By default, Renovate skips versions in between, like `python@v3.10.x`.
 
 But if you set `separateMultipleMinor=true` then you get separate PRs for each minor stream, like `python@3.9.x`, `python@v3.10.x` and `python@v3.11.x`, etc.
 
+## skipArtifactUpdating
+
+Use this option in cases where automatic artifact updating by Renovate fails or is incorrect, or you have custom needs.
```

Comment:
```suggestion
Use this option when automatic artifact updating fails, is incorrect, or not needed.  
This is useful if you have custom workflows handling artifact updates.

```

Reply:
@rarkins Do you want this change, or stick with what you previously suggested?
https://github.com/renovatebot/renovate/pull/35993/files/98a1e917c59b34176c3795abcdb834d9a95c6b6d#r2105764636